assemblyscript/examples/object-classification.ts

```diff
@@ -17,7 +17,8 @@ export function main(): i32 {
 
     Console.log("Running classification...");
     context.compute();
-    const output = context.getOutput(0);
+    let maxBufferLength = 4004; // Size of our output buffer
+    const output = context.getOutput(0, new Array<u8>(maxBufferLength).fill(0));
```

Comment:
```suggestion
    const output = context.getOutput(0, new Array<u8>(4004).fill(0));
```

We could just inline it, I guess.

Reply:
Could go either way, I only broke it out so that it was more obvious what the number 4004 was.

Reply:
I think I'll just go with the inline, since in the long run we plan on removing the need to do this at all.